modules/apps/27-interchain-accounts/host/client/cli/tx.go

```diff
@@ -0,0 +1,75 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/spf13/cobra"
+
+	icatypes "github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/types"
+)
+
+const (
+	memoFlag string = "memo"
+)
+
+func generatePacketDataCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "generate-packet-data [message]",
+		Short: "Generates ICA packet data.",
+		Long: fmt.Sprintf(`generate-packet-data accepts a message string and serializes it
+into packet data which is outputted to stdout. It can be used in conjunction with "%s ica controller send-tx"
```

Comment:
The binary might actually be different. The host might be on osmosis and the controller might be the hub. Maybe just mention it can be used in conjunction with the controller send-tx?

Reply:
Ah good point, yes I'll update the message!